### ISSUE TITLE
Bug introduced when going from 2.0.3 to 2.1.1

### ISSUE THREAD

**Issue by jcollum-nutrien** [ashiina/lambda-local#237]
Haven't fully tracked this down yet but I have a lambda local that I've been using reliably for months. Starts like this: 

```
lambda-local -l index.js -h handler --envfile ../.env --watch --inspect
``` 

No issues. But we updated to 2.1.1 the other day and a working payload that looks like 

```
{
  "event": {
    "action": "load",
    "payload": {
      "record": {        
        "name": "name_for_id_865dc2f8-f379-11e3-ae2b-3176bd53680f",      
        "sourceTimestamp": "2023-07-25T00:30:10.132Z"      
      },
      "messageId": "eaf60424-948c-4aa9-84bb-511bc59f6852"
    }
  }
}
```

is now giving me Zod errors for missing `action` property. But if I revert back to 2.0.3 that's not happening anymore. Was there a breaking change introduced in 2.1.1 ?

**Comment by gpotter2** [comment]
Hi. The default behavior changed, but maybe it shouldn't have.

What aws product/lambda are you trying to emulate? The new output is what AWS Gateway output.

Since there are no unit tests nor any comment of what the previous output was representing, this wasn't reported.

**Comment by gpotter2** [comment]
You're right this is a regression since it isn't tested. Will provide a fix soon.

**Comment by gpotter2** [comment]
Fixed in 2.1.2.
